src/routes/productRoutes.js

```diff
@@ -1,10 +1,15 @@
-const {Router} = require("express")
-const productController = require("../controllers/productController")
-
-
-const router = Router();
-
-
-router.post("/" , productController.createProduct)
-
-module.exports = router
\ No newline at end of file
+const { Router } = require('express')
+const productController = require('../controllers/productController')
+const {
+  getAllProducts,
+  getProduct,
+  deleteProduct,
+} = require('../services/productService')
+
+const router = Router()
+
+router.post('/', productController.createProduct)
+router.get('/', getAllProducts)
```

Comment:
as we said in the next pull request you should refactor the code to include the business logic in the productService no in productController